src/quo2/components/icons/icons.clj

```diff
@@ -41,4 +41,5 @@
    (get-icons 16)
    (get-icons 20)
    (get-icons 24)
-   (get-icons 32)))
+   (get-icons 32)
+   (get-icons 48)))
```

Comment:
Hi, we don't have 48 size variant in the [iconset](https://www.figma.com/file/qLLuMLfpGxK9OfpIavwsmK/Iconset?node-id=953%3A7&t=PsRlxrpkAliN7lmV-0).

Also, We already have the above imported icons in the iconset, So please remove them.
(Looks like for some reason, 3 dot's name is the `option` for 20x20 and `more` for other sizes).
So, Please use the icon with the name `option` and `keycard` from the 20x20 variant.

Reply:
Hey! Sorry I don't have access to the iconset link. 

About the 48x48. I added it beacuase the Ring in the `user-avatar` was needed for 48x48. 
About the more, the icon was missing from the 20x20 that's why I added it. I try using `i/:option` but it doesn't display. 
And about keycard I tried finding it but they keycard is not the same icon, it is different.

What should I do? We can chat over discord if you want about this since it's not clear to me what to do. `mattgle.eth#1997`

Reply:
I've messaged you over Discord about this